CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 * Add support for uap10.0.19041
 * Add support for Android 12
 * Support for Android 11 (March, 2021)
+* Support for NET 6
```

Comment:
need to add the versions in the changelog
and add removal of android 11 support

Reply:
Like I said in my other comment, there was no removal of android 11 anymore, because we only update dthe Uno version of the package WinUI, this PR should only make significant changes for the new WinUI package

Reply:
Got it for android11.
We still need to update the changelog to reflect the version number

Reply:
it should look like
## [0.5.0] - 2023-05-24
### Added
Support for NET 6
Support for WebView2 in the WinUI package

## [0.4.0]

Reply:
It should look like this

## [0.5.0] - 2023-05-24
### Added
* [#28]Support for NET 6
* [#28]Support for WebView2 in the WinUI package

## [0.4.0]